devfiles/lombok/devfile.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: javalombok-
+projects:
+  - name: lombok-demo
+    source:
+      location: 'https://github.com/che-samples/lombok-demo.git'
+      type: git
+      branch: main
+components:
+  - id: redhat/java/latest
+    type: chePlugin
+    volumes:
+      - containerPath: /tmp/vscode-unpacked
+        name: lombok-java-unpacked
+    preferences:
+      java.jdt.ls.vmargs: -javaagent:/tmp/vscode-unpacked/lombok.jar
+  - type: chePlugin
+    id: cool2k/vscode-lombok/latest
+    volumes:
+      - containerPath: /tmp/vscode-unpacked
+        name: lombok-java-unpacked
+    alias: vscode-lombok
+  - id: eclipse/che-theia/next
+    type: cheEditor
+  - type: dockerimage
+    command:
+      - sh
+      - -c
+    args:
+      - until [ -d /tmp/vscode-unpacked/*lombok* ]; do sleep 1; done; cp $(find /tmp |grep lombok |grep jar) /tmp/vscode-unpacked/; sleep infinity
```

Comment:
So that works but i don't think this is the right approach. It should be done by the Che plugin (entrypoint maybe ?)